protocol/communities/persistence.go

```diff
@@ -327,6 +327,17 @@ func (p *Persistence) rowsToCommunities(rows *sql.Rows) (comms []*Community, err
 	return comms, nil
 }
 
+func (p *Persistence) LeftCommunities(memberIdentity *ecdsa.PublicKey) (comms []*Community, err error) {
+	query := communitiesBaseQuery + ` WHERE NOT c.Joined`
+
+	rows, err := p.db.Query(query, common.PubkeyToHex(memberIdentity), RequestToJoinStatePending)
```

Comment:
is `r.state = ?` needed in query statement for `RequestToJoinStatePending`?

Reply:
nice catch, thanks. Solved